### DISCUSSION TITLE
4mmc + alcohol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#83] [Drug Interactions]
Seems similar to alcohol + other stimulants/amphetamines. Alcohol can decrease some of the stimulation without inhibiting the euphoria and the 4mmc reduces the intoxicating effects of alcohol. Some studies indicate that this can cause people to use more of both than they normally would. Unsure of how to classify this. Caution? Safe?

Berquist, M. D. I., Peet, M. M., & Baker, L. E. (2015). Behavioral sensitization following concurrent exposure to mephedrone and D-amphetamine in female mice. Behavioural Pharmacology, 26(1 and 2-Special Issue), 180. https://doi.org/10.1097/FBP.0000000000000121

Farré, M., Perez-Maña, C., Souza, E. de, Mateus, J., Theunisen, E., Kuypers, K., Ramaekers, J., Fonseca, F., Torrens, M., Olesti, E., Torre, R. de la, & Papaseit, E. (2016). Interactions between mephedrone and alcohol in humans: Cardiovascular and subjective effects. European Psychiatry, 33(S1), S115–S115. https://doi.org/10.1016/j.eurpsy.2016.01.122

Pieprzyca, E., Skowronek, R., & Czekaj, P. (2022). Toxicological Analysis of Intoxications with Synthetic Cathinones. Journal of Analytical Toxicology, 46(7), 705–711. https://doi.org/10.1093/jat/bkab102

**Comment by utaninja** [comment]
We've got a discussion posted about this already, I'm going to forward this post there and lock this discussion just to keep track of everything :)
